__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const app = require("../app");
+const db = require("../db/connection");
+const testData = require("../db/data/test-data");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+ test("GET: should respond with a 200 status code", () => {
+  return request(app).get("/api/topics").expect(200);
+ });
+ test("should return an array of objects, all with correct properties", () => {
+  return request(app)
+   .get("/api/topics")
+   .then(({ body }) => {
+    expect(body.topics.length).toBe(3);
+    body.topics.forEach((topic) => {
+     expect(typeof topic.description).toBe("string");
+     expect(typeof topic.slug).toBe("string");
```

Comment:
This is a good test, you're allowing for the potential expansion of more properties on this endpoint without this test breaking which is a good way to test. 
I'd recommend checking out `.toMatchObject` as a matcher which is tailored to do this type of assertion